### PR TITLE
EOS-10183: Add logging to m0_be_recovery_run()

### PR DESCRIPTION
**Pull request by suvratjoshi**
Add logging to function m0_be_recovery_run().
Look at the function code and add M0_WARN logging for all pointers,
movements, actions, decisions.

### CODE REVIEW

File: be/recovery.c [hunk 1]
```diff
@@ -192,41 +192,53 @@ M0_INTERNAL int m0_be_recovery_run(struct m0_be_recovery *rvr)
...
 		prev = iter;
+		M0_LOG(M0_DEBUG, "Forward movement : prev header "
+				  BFLRH_F, BFLRH_P(&prev->lri_header));
 		rc = be_recovery_log_record_iter_new(&iter);
 		rc = rc ?: m0_be_log_record_next(log, prev, iter);
 	}
 	be_recovery_log_record_iter_destroy(iter);
-	if (!M0_IN(rc, (0, -ENOENT)))
+	if (!M0_IN(rc, (0, -ENOENT))){
+		M0_LOG(M0_ERROR, "Error");
```
Comment (huanghua78) [inline be/recovery.c:217]: M0_LOG(M0_ERROR, "Error: rc=%d", rc);

File: be/recovery.c [hunk 2]
```diff
@@ -235,13 +247,17 @@ M0_INTERNAL int m0_be_recovery_run(struct m0_be_recovery *rvr)
...
+			M0_LOG(M0_DEBUG, "Backward movement : prev header "
+					  BFLRH_F, BFLRH_P(&(prev->lri_header)));
 		} else
 			be_recovery_log_record_iter_destroy(iter);
 	}
 	M0_ASSERT(ergo(rc == 0, prev->lri_header.lrh_pos == last_discarded));
 
-	if (rc != 0)
+	if (rc != 0){
+		M0_LOG(M0_ERROR, "Error");
```
Comment (huanghua78) [inline be/recovery.c:258]: M0_LOG(M0_ERROR, "Error: rc=%d", rc);

File: be/recovery.c (hunk 1 above)
Comment (suvratjoshi) [reply be/recovery.c]: Done!

File: be/recovery.c (hunk 2 above)
Comment (suvratjoshi) [reply be/recovery.c]: Done!

**Review by huanghua78 — APPROVED** [review]

**Comment by cortx-admin** [comment]
<h1>Jenkins CI Result : <a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/31/>Motr#31</a></h1><h2>Motr Test Summary</h2><table><tr><th>Test Result</th><th>Count</th><th>Info</tr><tr><td>:x:Failed</td><td>2</td><td><details><summary>:file_folder:</summary><p><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/31//testReport/junit/motr-single-node/47motr-tests-user-kernel target=_blank>47motr-tests-user-kernel</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/31//testReport/junit/motr-single-node/32m0t1fs-rconfc-fail-test target=_blank>32m0t1fs-rconfc-fail-test</a></p></details></td></tr><tr><td>:checkered_flag:Skipped</td><td>5</td><td><details><summary>:file_folder:</summary><p><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/31//testReport/junit/motr-single-node/31motr-sys-kvs-kernel-test target=_blank>31motr-sys-kvs-kernel-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/31//testReport/junit/motr-single-node/07multi-client-tests target=_blank>07multi-client-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/31//testReport/junit/motr-single-node/39mount-fail target=_blank>39mount-fail</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/31//testReport/junit/motr-single-node/12sns-cc-io-tests target=_blank>12sns-cc-io-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/31//testReport/junit/motr-single-node/37protocol target=_blank>37protocol</a></p></details></td></tr><tr><td>:heavy_check_mark:Passed</td><td>55</td><td><details><summary>:file_folder:</summary><p><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/31//testReport/junit/motr-single-node/40motr-dgmode target=_blank>40motr-dgmode</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/31//testReport/junit/motr-single-node/51motr-rmw target=_blank>51motr-rmw</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/31//testReport/junit/motr-single-node/50motr-rm-lock-cc-io target=_blank>50motr-rm-lock-cc-io</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/31//testReport/junit/motr-single-node/20pool-version-test target=_blank>20pool-version-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/31//testReport/junit/motr-single-node/46m0crate target=_blank>46m0crate</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/31//testReport/junit/motr-single-node/54sns-repair-motr-1f target=_blank>54sns-repair-motr-1f</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/31//testReport/junit/motr-single-node/25spiel-sns-repair-test target=_blank>25spiel-sns-repair-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/31//testReport/junit/motr-single-node/21spiel-test target=_blank>21spiel-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/31//testReport/junit/motr-single-node/59sns-repair-motr-abort target=_blank>59sns-repair-motr-abort</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/31//testReport/junit/motr-single-node/01kernel-tests target=_blank>01kernel-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/31//testReport/junit/motr-single-node/33motr-st target=_blank>33motr-st</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/31//testReport/junit/motr-single-node/42motr-utils target=_blank>42motr-utils</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/31//testReport/junit/motr-single-node/19sns-abort-test target=_blank>19sns-abort-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/31//testReport/junit/motr-single-node/09poolmach-tests target=_blank>09poolmach-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/31//testReport/junit/motr-single-node/13sns-repair-quiesce target=_blank>13sns-repair-quiesce</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/31//testReport/junit/motr-single-node/41motr-conf-update target=_blank>41motr-conf-update</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/31//testReport/junit/motr-single-node/17sss-st-test target=_blank>17sss-st-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/31//testReport/junit/motr-single-node/00userspace-tests target=_blank>00userspace-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/31//testReport/junit/motr-single-node/03initscript-tests target=_blank>03initscript-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/31//testReport/junit/motr-single-node/45motr-sns-repair-N-1 target=_blank>45motr-sns-repair-N-1</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/31//testReport/junit/motr-single-node/30hagen-test target=_blank>30hagen-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/31//testReport/junit/motr-single-node/11sns-multi-tests target=_blank>11sns-multi-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/31//testReport/junit/motr-single-node/58sns-repair-motr-ios-fail target=_blank>58sns-repair-motr-ios-fail</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/31//testReport/junit/motr-single-node/29confgen-test target=_blank>29confgen-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/31//testReport/junit/motr-single-node/34sns-repair-1n-1f target=_blank>34sns-repair-1n-1f</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/31//testReport/junit/motr-single-node/44motr-sns-repair target=_blank>44motr-sns-repair</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/31//testReport/junit/motr-single-node/08degraded-mode-tests target=_blank>08degraded-mode-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/31//testReport/junit/motr-single-node/26spiel-sns-repair-quiesce-test target=_blank>26spiel-sns-repair-quiesce-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/31//testReport/junit/motr-single-node/60sns-repair-motr-abort-quiesce target=_blank>60sns-repair-motr-abort-quiesce</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/31//testReport/junit/motr-single-node/35m0singlenode target=_blank>35m0singlenode</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/31//testReport/junit/motr-single-node/14m0t1fs-fsync-test target=_blank>14m0t1fs-fsync-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/31//testReport/junit/motr-single-node/43motr-sync-replication target=_blank>43motr-sync-replication</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/31//testReport/junit/motr-single-node/04net-tests target=_blank>04net-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/31//testReport/junit/motr-single-node/55sns-repair-motr-1n-1f target=_blank>55sns-repair-motr-1n-1f</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/31//testReport/junit/motr-single-node/24m0d-fsync-test target=_blank>24m0d-fsync-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/31//testReport/junit/motr-single-node/49motr-rpc-cancel target=_blank>49motr-rpc-cancel</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/31//testReport/junit/motr-single-node/56sns-repair-motr-mf target=_blank>56sns-repair-motr-mf</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/31//testReport/junit/motr-single-node/38sns-abort-quiesce-test target=_blank>38sns-abort-quiesce-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/31//testReport/junit/motr-single-node/15m0t1fs-fwait-test target=_blank>15m0t1fs-fwait-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/31//testReport/junit/motr-single-node/02system-tests target=_blank>02system-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/31//testReport/junit/motr-single-node/22rpc-cancel-test target=_blank>22rpc-cancel-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/31//testReport/junit/motr-single-node/53clusterusage-alert target=_blank>53clusterusage-alert</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/31//testReport/junit/motr-single-node/05rpcping-tests target=_blank>05rpcping-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/31//testReport/junit/motr-single-node/23m0d-signal-test target=_blank>23m0d-signal-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/31//testReport/junit/motr-single-node/28motr-sys-kvs-test target=_blank>28motr-sys-kvs-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/31//testReport/junit/motr-single-node/06console-tests target=_blank>06console-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/31//testReport/junit/motr-single-node/18m0mount-test target=_blank>18m0mount-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/31//testReport/junit/motr-single-node/27sns-repair-io-fail-test target=_blank>27sns-repair-io-fail-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/31//testReport/junit/motr-single-node/16conf-st-test target=_blank>16conf-st-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/31//testReport/junit/motr-single-node/52singlenode-sanity target=_blank>52singlenode-sanity</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/31//testReport/junit/motr-single-node/57sns-repair-motr-1k-1f target=_blank>57sns-repair-motr-1k-1f</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/31//testReport/junit/motr-single-node/48motr-raid0-io target=_blank>48motr-raid0-io</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/31//testReport/junit/motr-single-node/10sns-single-tests target=_blank>10sns-single-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/31//testReport/junit/motr-single-node/36spare-reservation target=_blank>36spare-reservation</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/31//testReport/junit/motr-single-node/51kem target=_blank>51kem</a></p></details></td></tr><tr><td>Total</td><td>62</td><td><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/31//testReport/junit/motr-single-node>:link:</a></td></tr></table><h2> CppCheck Summary</h2><p> &emsp;&emsp;&emsp;Cppcheck: No new warnings found :thumbsup:</p>

**Comment by huanghua78** [comment]
@suvratjoshi 
please check if this PR is still valid and needed.
If not, please close it.
If yes, please update the status.

**Comment by suvratjoshi** [comment]
hi @huanghua78 
This task was created while resolution of data corruption issues was in progress. Its beneficial to have logs for sure I think, but I don't know if it is still needed. It was kept open for review from @max-seagate

**Comment by huanghua78** [comment]
> hi @huanghua78
> This task was created while resolution of data corruption issues was in progress. Its beneficial to have logs for sure I think, but I don't know if it is still needed. It was kept open for review from @max-seagate

Thank you, @suvratjoshi .

Hello @madhavemuri , @mukundkanekar , @nihar1313 , can you please have a review for this PR?
It's simple and just adding some log messages.

**Review by mukundkanekar — APPROVED** [review]

**Comment by cortx-admin** [comment]
<h1>Jenkins CI Result : <a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/238/>Motr#238</a></h1><h2>Motr Test Summary</h2><table><tr><th>Test Result</th><th>Count</th><th>Info</tr><tr><td>:x:Failed</td><td>1</td><td><details><summary>:file_folder:</summary><p><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/238//testReport/junit/04motr-single-node/49motr-rpc-cancel target=_blank>04motr-single-node/49motr-rpc-cancel</a></p></details></td></tr><tr><td>:checkered_flag:Skipped</td><td>5</td><td><details><summary>:file_folder:</summary><p><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/238//testReport/junit/01motr-single-node/37protocol target=_blank>01motr-single-node/37protocol</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/238//testReport/junit/04motr-single-node/31motr-sys-kvs-kernel-test target=_blank>04motr-single-node/31motr-sys-kvs-kernel-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/238//testReport/junit/05motr-single-node/07multi-client-tests target=_blank>05motr-single-node/07multi-client-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/238//testReport/junit/05motr-single-node/39mount-fail target=_blank>05motr-single-node/39mount-fail</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/238//testReport/junit/05motr-single-node/12sns-cc-io-tests target=_blank>05motr-single-node/12sns-cc-io-tests</a></p></details></td></tr><tr><td>:heavy_check_mark:Passed</td><td>58</td><td><details><summary>:file_folder:</summary><p><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/238//testReport/junit/01motr-single-node/46m0crate target=_blank>01motr-single-node/46m0crate</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/238//testReport/junit/01motr-single-node/01kernel-tests target=_blank>01motr-single-node/01kernel-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/238//testReport/junit/01motr-single-node/00userspace-tests target=_blank>01motr-single-node/00userspace-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/238//testReport/junit/01motr-single-node/03initscript-tests target=_blank>01motr-single-node/03initscript-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/238//testReport/junit/01motr-single-node/30hagen-test target=_blank>01motr-single-node/30hagen-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/238//testReport/junit/01motr-single-node/29confgen-test target=_blank>01motr-single-node/29confgen-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/238//testReport/junit/01motr-single-node/35m0singlenode target=_blank>01motr-single-node/35m0singlenode</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/238//testReport/junit/01motr-single-node/04net-tests target=_blank>01motr-single-node/04net-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/238//testReport/junit/01motr-single-node/05rpcping-tests target=_blank>01motr-single-node/05rpcping-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/238//testReport/junit/01motr-single-node/28motr-sys-kvs-test target=_blank>01motr-single-node/28motr-sys-kvs-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/238//testReport/junit/01motr-single-node/06console-tests target=_blank>01motr-single-node/06console-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/238//testReport/junit/01motr-single-node/52singlenode-sanity target=_blank>01motr-single-node/52singlenode-sanity</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/238//testReport/junit/02motr-single-node/20pool-version-test target=_blank>02motr-single-node/20pool-version-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/238//testReport/junit/02motr-single-node/21spiel-test target=_blank>02motr-single-node/21spiel-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/238//testReport/junit/02motr-single-node/47motr-tests-user-kernel target=_blank>02motr-single-node/47motr-tests-user-kernel</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/238//testReport/junit/02motr-single-node/09poolmach-tests target=_blank>02motr-single-node/09poolmach-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/238//testReport/junit/02motr-single-node/41motr-conf-update target=_blank>02motr-single-node/41motr-conf-update</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/238//testReport/junit/02motr-single-node/08degraded-mode-tests target=_blank>02motr-single-node/08degraded-mode-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/238//testReport/junit/02motr-single-node/24m0d-fsync-test target=_blank>02motr-single-node/24m0d-fsync-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/238//testReport/junit/02motr-single-node/02system-tests target=_blank>02motr-single-node/02system-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/238//testReport/junit/02motr-single-node/22rpc-cancel-test target=_blank>02motr-single-node/22rpc-cancel-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/238//testReport/junit/02motr-single-node/53clusterusage-alert target=_blank>02motr-single-node/53clusterusage-alert</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/238//testReport/junit/02motr-single-node/23m0d-signal-test target=_blank>02motr-single-node/23m0d-signal-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/238//testReport/junit/02motr-single-node/51kem target=_blank>02motr-single-node/51kem</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/238//testReport/junit/03motr-single-node/54sns-repair-motr-1f target=_blank>03motr-single-node/54sns-repair-motr-1f</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/238//testReport/junit/03motr-single-node/59sns-repair-motr-abort target=_blank>03motr-single-node/59sns-repair-motr-abort</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/238//testReport/junit/03motr-single-node/17sss-st-test target=_blank>03motr-single-node/17sss-st-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/238//testReport/junit/03motr-single-node/58sns-repair-motr-ios-fail target=_blank>03motr-single-node/58sns-repair-motr-ios-fail</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/238//testReport/junit/03motr-single-node/60sns-repair-motr-abort-quiesce target=_blank>03motr-single-node/60sns-repair-motr-abort-quiesce</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/238//testReport/junit/03motr-single-node/55sns-repair-motr-1n-1f target=_blank>03motr-single-node/55sns-repair-motr-1n-1f</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/238//testReport/junit/03motr-single-node/56sns-repair-motr-mf target=_blank>03motr-single-node/56sns-repair-motr-mf</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/238//testReport/junit/03motr-single-node/61spiel-multi-confd target=_blank>03motr-single-node/61spiel-multi-confd</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/238//testReport/junit/03motr-single-node/16conf-st-test target=_blank>03motr-single-node/16conf-st-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/238//testReport/junit/03motr-single-node/57sns-repair-motr-1k-1f target=_blank>03motr-single-node/57sns-repair-motr-1k-1f</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/238//testReport/junit/03motr-single-node/36spare-reservation target=_blank>03motr-single-node/36spare-reservation</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/238//testReport/junit/04motr-single-node/51motr-rmw target=_blank>04motr-single-node/51motr-rmw</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/238//testReport/junit/04motr-single-node/50motr-rm-lock-cc-io target=_blank>04motr-single-node/50motr-rm-lock-cc-io</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/238//testReport/junit/04motr-single-node/25spiel-sns-repair-test target=_blank>04motr-single-node/25spiel-sns-repair-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/238//testReport/junit/04motr-single-node/33motr-st target=_blank>04motr-single-node/33motr-st</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/238//testReport/junit/04motr-single-node/19sns-abort-test target=_blank>04motr-single-node/19sns-abort-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/238//testReport/junit/04motr-single-node/32m0t1fs-rconfc-fail-test target=_blank>04motr-single-node/32m0t1fs-rconfc-fail-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/238//testReport/junit/04motr-single-node/34sns-repair-1n-1f target=_blank>04motr-single-node/34sns-repair-1n-1f</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/238//testReport/junit/04motr-single-node/26spiel-sns-repair-quiesce-test target=_blank>04motr-single-node/26spiel-sns-repair-quiesce-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/238//testReport/junit/04motr-single-node/27sns-repair-io-fail-test target=_blank>04motr-single-node/27sns-repair-io-fail-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/238//testReport/junit/04motr-single-node/48motr-raid0-io target=_blank>04motr-single-node/48motr-raid0-io</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/238//testReport/junit/05motr-single-node/40motr-dgmode target=_blank>05motr-single-node/40motr-dgmode</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/238//testReport/junit/05motr-single-node/42motr-utils target=_blank>05motr-single-node/42motr-utils</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/238//testReport/junit/05motr-single-node/13sns-repair-quiesce target=_blank>05motr-single-node/13sns-repair-quiesce</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/238//testReport/junit/05motr-single-node/45motr-sns-repair-N-1 target=_blank>05motr-single-node/45motr-sns-repair-N-1</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/238//testReport/junit/05motr-single-node/11sns-multi-tests target=_blank>05motr-single-node/11sns-multi-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/238//testReport/junit/05motr-single-node/44motr-sns-repair target=_blank>05motr-single-node/44motr-sns-repair</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/238//testReport/junit/05motr-single-node/14m0t1fs-fsync-test target=_blank>05motr-single-node/14m0t1fs-fsync-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/238//testReport/junit/05motr-single-node/43motr-sync-replication target=_blank>05motr-single-node/43motr-sync-replication</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/238//testReport/junit/05motr-single-node/38sns-abort-quiesce-test target=_blank>05motr-single-node/38sns-abort-quiesce-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/238//testReport/junit/05motr-single-node/15m0t1fs-fwait-test target=_blank>05motr-single-node/15m0t1fs-fwait-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/238//testReport/junit/05motr-single-node/18m0mount-test target=_blank>05motr-single-node/18m0mount-test</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/238//testReport/junit/05motr-single-node/10sns-single-tests target=_blank>05motr-single-node/10sns-single-tests</a></br><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/238//testReport/junit/motr-single-node/62dix-repair-st-lookup-insert target=_blank>motr-single-node/62dix-repair-st-lookup-insert</a></p></details></td></tr><tr><td>Total</td><td>64</td><td><a href=http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Motr/238//testReport/junit>:link:</a></td></tr></table><h2> CppCheck Summary</h2><p> &emsp;&emsp;&emsp;Cppcheck: No new warnings found :thumbsup:</p>

**Pull request merged**